### ISSUE TITLE
SCSS: double slashes in comments are split 

### ISSUE THREAD

**Issue by jnns** [prettier/prettier#7647]
**Prettier 1.19.1**
[Playground link](https://prettier.io/playground/#.....)
Another issue with SCSS comments where the `//` is split into `/ /`:

**Input** 
```scss
.real-world-example {
  background: radial-gradient(
      circle at left 0% bottom $position,
      transparent,
      transparent $diameter,
      #fbfbfb calc(#{$diameter} + 1px) // Add 1px for edge-smoothing.
    );
}

.simplification { 
  foo: (
    calc() // not a comment anymore
  );
}
```
**Output** ([Playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEA6ATnAhgGwLQDuE6OAJnnAB5YC2ADjnAATAA6UTTARlmANYBzdBACuUUkibospAJa48QmbIQwAFO05amYWejCMmWGE0YAzEwAYApNwgwYEGkwAkdCAGdZMWdAA0mtpMMNJQHnRYmLABHEEhWGERUSYucrRw8OgxQUwAxGZcBQU6uGBqucCp8jQZcOgAvkwA1EwAjHSUAJRMAPQ9TACCpKRtHUxmxExwpAJweB40EPYAFrJQAqiBnJ0A3Oz17OyoXvQ4smayYMa+HMBMgRMQkhqxnFc4Zd19TFD2RjpOGqwIxQACei0whxgu327BAfhAEDoPmgHmQoEiwgIAAVIgg0SgsAA3CCyUjwkBcaT8DIAZQiunWyBCIjgCOWMBoOAA6qt4OFeHBafjvLIid5QchwB40Qi1h46jBsdIBDQsMgzLgFQiAFYeSgAIWpfDp6QAMms4BqtWyQHrKLS1gJGABFET2K1ITU4bUgJIK9BSjxgGUUujoNYwblkmDLZAADksCPDnjg3OkdCl4bgAaJVoRAEd3fBlUiCSAsB48FA4NNphTMEW9HBlVhVeqvTaEQqaLJmehWd2na7i57vb6YFguNHSLHkAAmBHxWRndYAYUBHZAvxrFJECoAKlOCePbUTWQBJcSqWlgCPIgbiWkwUGMa0+uD1epAA))

```scss
.real-world-example {
  background: radial-gradient(
    circle at left 0% bottom $position,
    transparent,
    transparent $diameter,
    #fbfbfb calc(#{$diameter} + 1px) / / Add 1px for edge-smoothing.
  );
}

.simplification {
  foo: (calc() / / not a comment anymore);
}
```
**Expected output** 

```scss
.real-world-example {
  background: radial-gradient(
    circle at left 0% bottom $position,
    transparent,
    transparent $diameter,
    #fbfbfb calc(#{$diameter} + 1px) // Add 1px for edge-smoothing.
  );
}

.simplification {
  foo: (calc() // not a comment anymore);
}
```

**Comment by mohitgogna** [comment]
Is this bug still open?

**Comment by fisker** [comment]
Seems fixed somehow, need confirm.

**Prettier 2.0.4**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEA6ATnAhgGwLQDuE6OAJnnAB5YC2ADjnAATAA6UTTARlmANYBzdBACuUUkibospAJa48QmbIQwAFO05amYWejCMmWGE0YAzEwAYApNwgwYEGkwAkdCAGdZMWdAA0mtpMMNJQHnRYmLABHEEhWGERUSYucrRw8OgxQUwAxGZcBQU6uGBqucCp8jQZcOgAvkwA1EwAjHSUAJRMAPQ9TACCpKRtHUxmxExwpAJweB40EPYAFrJQAqiBnJ0A3Oz17OyoXvQ4smayYMa+HMBMgRMQkhqxnFc4Zd19TFD2RjpOGqwIxQACei0wgV2+xAfhAEDoPmgHmQoEiwgIAAVIggUShcAQsKCUXCuNJ+BkAMoRXTrZAhERwOHLGA0HAAdVW8HCvDglNx3lkADdvKDkOAPCSQGsPHUYJjpAIaFhkGZcLK4QArDyUABC5L4VPSABk1nBVeqmSBtZRKWsBIwAIoiezmpBqnAakBJWXocUeMCS2He9BrGDs2SkGDLZAADkscLowll7OkdHFSbgvqF5rhAEcXfAFQi8SAsB48FA4NNpsHMAW9HAFVglSr3Za4bKaLJ6ehGZ37U7C26PV6YFguBGozGkAAmOHxWRndYAYUBbZAWYArMGRLKACoTvGjq1CxkASXEqkpYFDiIG4kpMFBjAtnrg9XqQA)
```sh
--parser scss
```

**Input:**
```scss
.real-world-example {
  background: radial-gradient(
      circle at left 0% bottom $position,
      transparent,
      transparent $diameter,
      #fbfbfb calc(#{$diameter} + 1px) // Add 1px for edge-smoothing.
    );
}

.simplification { 
  foo: (
    calc() // not a comment anymore
  );
}
```

**Output:**
```scss
.real-world-example {
  background: radial-gradient(
    circle at left 0% bottom $position,
    transparent,
    transparent $diameter,
    #fbfbfb calc(#{$diameter} + 1px) // Add 1px for edge-smoothing.
  );
}

.simplification {
  foo: (calc() // not a comment anymore);
}

```

**Comment by jnns** [comment]
Thank you for looking into this @fisker. It really seems to be fixed on 2.0.4 at least. I didn't try with previous versions. 

You rightfully point out that there's a second bug: lines containing a double-slash comment shouldn't be joined carelessly. 

The current behaviour matches the expected behaviour in my initial post because I just adjusted the broken output by hand and didn't notice that it still isn't valid markup with the initial bug fixed.

As this bug's description clearly only deals with the unnecessary space between the two slashes, I'd vote for closing this bug and opening a new one about broken line-joining for lines with double-slash comments in them.
